holidays/countries/indonesia.py

```
@@ -255,187 +255,81 @@ class IndonesiaChineseHolidays(_CustomChineseHolidays):
+        2001: (MAR, 5),
         2004: (FEB, 2),
-        2010: (NOV, 17),
-        2014: (OCT, 5),
-        2015: (SEP, 24),
-        2016: (SEP, 12),
-        2018: (AUG, 22),
-        2022: (JUL, 10),
-        2023: (JUN, 29),
-        2024: (JUN, 17),
+        2025: (JUN, 6),
     }
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)